integrations/jenkins/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,35 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: jenkinsJob
+    selector:
+      query:  "true"
+    port:
+      entity:
+        mappings:
+          identifier: 'if .url | contains("://") then (.url | split("://")[1] | sub("^.*?/"; "")) else .url end | sub("%20"; "-"; "g") | sub("/"; "-"; "g") | .[:-1]'
+          title: .displayName
+          blueprint: '"jenkinsJob"'
+          properties:
+            jobName: .fullName
+            url: '.url | if test("://") then sub("^[^/]+//[^/]+"; "") else . end'
+            jobStatus: '.color as $input | if $input == "notbuilt" then "created" else "updated" end'
+            timestamp: .time
+          relations:
+            jobs: '.jobs | map(if .url | contains("://") then (.url | split("://")[1] | sub("^.*?/"; "")) else .url end | sub("%20"; "-"; "g") | sub("/"; "-"; "g") | .[:-1])'
+  - kind: jenkinsBuild
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: '.fullDisplayName | sub(" "; "-"; "g") | sub("#"; ""; "g") | sub("»"; ""; "g") '
```

Comment:
please use the identifiers display names might change

Reply:
so we now have a `__jobName` field